src/gp_doctest.cpp

```diff
@@ -141,19 +141,6 @@ TEST_CASE("GPInstance: gradient calculation") {
   CHECK_LT(fabs(log_lik_and_derivative.second - -0.6109379521), 1e-6);
 }
 
-// Regression test.
```

Comment:
It's always sad to lose a test. Is this not relevant anymore?

Reply:
Sort of. Regression test that involves branch length is prevalent in another test (https://github.com/phylovi/libsbn/blob/4eabe4405edcc56e82c94e0a701cd8baa30fd863/src/gp_doctest.cpp#L215-L222).

 We can certainly add it back. Let me know what you think.

Reply:
If there's not a compelling reason to take it out, let's leave it.